crates/collateral-contract/script/query.sh

```
@@ -0,0 +1,10 @@
+# basic query to verify the contract is deployed and initialized
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)